src/validation.rs

```diff
@@ -708,6 +718,15 @@ const GLOBAL_EXTENSIONS_PYTHON_3_12: &[&str] = &[
     "_zoneinfo",
 ];
 
+const GLOBAL_EXTENSIONS_PYTHON_3_13: &[&str] = &[
+    "_sha2",
+    "_tokenize",
+    "_typing",
+    "_xxinterpchannels",
```

Comment:
These will need to be removed, and possibly some others added here instead (at a minimum the non-'xx' versions of these same modules).

Reply:
I've left a FIXME in the code for this to be resolved later.